designs/titan/src/front.js

```diff
@@ -295,15 +295,78 @@ export default (part) => {
+	if (points.seatMid.y > points.crotchSeamCurveStartMid.y){
+	points.seatIn = utils.lineIntersectsCurve(points.seatMid, points.seatInTarget, points.fork, points.crotchSeamCurveCp1, points.crotchSeamCurveCp2, points.crotchSeamCurveStart)
+	}
+	else {
+	points.seatIn = utils.beamsIntersect(points.seatMid, points.seatInTarget, points.crotchSeamCurveStart, points.waistIn)
+	}
+	if (options.fitKnee) {
+	if (points.waistOut.x < points.seatOut.x) {
+	points.hipsOut = utils.lineIntersectsCurve(points.hipsOutTarget, points.hipsIn.rotate(180, points.hipsOutTarget), points.waistOut, points.seatOut, points.kneeOutCp1, points.kneeOut)
+	points.seatOutNotch = utils.lineIntersectsCurve(points.seatMid, points.seatOutTarget, points.waistOut, points.seatOut, points.kneeOutCp1, points.kneeOut)
+	}
+	else {
+	points.hipsOut = utils.lineIntersectsCurve(points.hipsOutTarget, points.hipsIn.rotate(180, points.hipsOutTarget), points.waistOut, points.waistOut, points.seatOutCp1, points.seatOut)
+	points.seatOutNotch = points.seatOut
+	}
+	points.kneeOutNotch = points.kneeOut
+	points.kneeInNotch = points.kneeIn
+	}
+	else {
+	if (points.waistOut.x < points.seatOut.x) {
+	points.hipsOut = utils.lineIntersectsCurve(points.hipsOutTarget, points.hipsIn.rotate(180, points.hipsOutTarget), points.waistOut, points.seatOut, points.kneeOutCp1, points.floorOut)
+	points.seatOutNotch = utils.lineIntersectsCurve(points.seatMid, points.seatOutTarget, points.waistOut, points.seatOut, points.kneeOutCp1, points.floorOut)
+	points.kneeOutNotch = utils.lineIntersectsCurve(points.kneeOut, points.kneeIn.rotate(180, points.kneeOut), points.waistOut, points.seatOut, points.kneeOutCp1, points.floorOut)
+	}
+	else {
+	points.hipsOut = utils.lineIntersectsCurve(points.hipsOutTarget, points.hipsIn.rotate(180, points.hipsOutTarget), points.waistOut, points.waistOut, points.seatOutCp1, points.seatOut)
+	points.seatOutNotch = points.seatOut
+	points.kneeOutNotch = utils.lineIntersectsCurve(points.kneeOut, points.kneeIn.rotate(180, points.kneeOut), points.seatOut, points.seatOutCp2, points.kneeOutCp1, points.floorOut)
+	}
+	points.kneeInNotch = utils.lineIntersectsCurve(points.kneeIn, points.kneeOut.rotate(180, points.kneeIn), points.floorIn, points.kneeInCp2, points.forkCp1, points.fork)
+	}
+	macro('sprinkle', {
+	snippet: 'notch',
+	on: ['crotchSeamCurveStart', 'seatIn', 'seatOutNotch', 'kneeInNotch', 'kneeOutNotch']
+	})
+	paths.seatline = new Path()
+	.move(points.seatOutNotch)
+	.line(points.seatIn)
+	.attr('class', 'fabric help')
+	.attr('data-text', 'Seat Line')
+	.attr('data-text-class', 'center')
+	if (measurements.waistToHips * (1 - options.waistHeight) + absoluteOptions.waistbandWidth < measurements.waistToHips){
+	macro('sprinkle', {
+	snippet: 'notch',
+	on: ['hipsIn', 'hipsOut',]
+	})
+	paths.hipline = new Path()
+	.move(points.hipsOut)
+	.line(points.hipsIn)
+	.attr('class', 'fabric help')
+	.attr('data-text', 'Hip Line')
+	.attr('data-text-class', 'center')
+	}
+}
```

Comment:
Similarly, in this whole section I found it very difficult to read the non-indented code to determine what the code was supposed to do. Please add indentation.

Reply:
I have made this changed but personally I find the opposite to be true.